packages/extension/src/view/devtools/components/cookies/cookiesListing/cookieTableContainer/index.tsx

```diff
@@ -128,11 +125,15 @@ const CookieTableContainer = ({
         ),
       },
       {
-        header: 'Cookie Accepted',
+        header: 'Cookie Affected',
```

Comment:
Do you want to keep the name as “Cookie Affected” only?

Reply:
Oh my bad, I wanted to revert my change of changing it to "Cookie Blocked". The method we are using currently doesn't truly detects if a cookie is blocked, it checks if its in the jar. There fore we should not rename it to "Cookie Blocked"